smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/DocumentShapeDeserVisitor.java

```diff
@@ -372,7 +372,7 @@ public final Void documentShape(DocumentShape shape) {
      * const deserializeAws_restJson1_1ParameterList = (
      *   output: any,
      *   context: SerdeContext
-     * ): Array<Parameter> => {
+     * ): Parameter[] => {
```

Comment:
There's a similar comment in `DocumentShapeSerVisitor` to update.

Reply:
Resolved it in https://github.com/awslabs/smithy-typescript/pull/168/commits/f1ebf36ae9519500aa7577b69ab8eba87b105232